src/sec_edgar_bulker/cli/main.py

```
@@ -0,0 +1,257 @@
+            if download_mode not in ['sequential', 'concurrent']:
+                raise click.BadParameter(
+                    f"Invalid value for '--download-mode': {download_mode}. "
+                    "Must be one of: sequential, concurrent"
+                )
+            config.download.mode = download_mode
+            
+        if years:
+            try:
+                year_list = [int(y.strip()) for y in years.split(',')]
+                # Validate years
+                current_year = datetime.now().year
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)